app/models/streaming_event.rb

```diff
@@ -1,22 +1,19 @@
 class StreamingEvent < ApplicationRecord
-  def self.live
-    where('starts_at <= NOW() and ends_at >= NOW()').first
-  end
+  scope :live, -> { where('starts_at <= NOW() AND ends_at >= NOW()') }
+  scope :scheduled, -> { where('starts_at > NOW()') }
+  scope :featured, -> { where(featured: true) }
 
-  def self.next_featured
-    where(featured: true).
-      where('starts_at > NOW()').
-      order(:starts_at).
-      first
-  end
+  def self.current_live = live.first
```

Comment:
This is the only bit I'd probably change. I'd probably remove this and just use `live.first` in the controller. Because remembering the different between "live" and "current_live" isn't intuitive to me without looking at the code, and as it's currently only used in one place.